frontend/packages/dev-console/src/components/pipelineruns/status/StatusPopoverContent.scss

```diff
@@ -0,0 +1,3 @@
+.odc-statuspopover-content {
+  min-height: 175px;
+}
```

Comment:
Understand that the content could not have a full dynamic high because of the left-arrow, but here is an recommendation that we could grow the code snippet to the available space. What do you think?

Before:
![image](https://user-images.githubusercontent.com/139310/98821357-bc87a080-242f-11eb-8796-e814e9d6964b.png)

With the following flex layout:
![image](https://user-images.githubusercontent.com/139310/98821422-d2956100-242f-11eb-831b-9b79e77a7cce.png)

```suggestion
.odc-statuspopover-content {
  min-height: 175px;
  display: flex;
  flex-direction: column;
  pre {
    flex-grow: 1;
  }
}
```